gui/qt/history_list.py

```diff
@@ -44,6 +44,17 @@
     "confirmed.png",
 ]
 
+class HistoryTreeWidgetItem(QTreeWidgetItem):
+    def __init__(self, parent=None):
+        QTreeWidgetItem.__init__(self, parent)
+    
+    def __lt__(self, otherItem):
+        column = self.treeWidget().sortColumn()
+        try:
```

Comment:
suggestion: adding a conditional for column 0 to check the tx status (or perhaps confirmation).  This way sorting column 0 does the expected thing and group tx's by their icon (or by their confirmations? Jonald? What do you think)?

Right now sorting column 0 does nothing since text is always ""

Sorting column zero also requires: You add a item.setData(0, Qt.UserRole+1, status) or perhaps setData(0, Qt.UserRole+1,conf) call in on_update()